buildlib/az-helpers.sh

```diff
@@ -111,3 +111,20 @@ function az_module_unload() {
     module=$1
     module unload "${module}" || true
 }
+
+
+#
+# try load cuda modules if nvidia driver is installed
+#
+try_load_cuda_env() {
```

Comment:
remove from test_jenkins.sh?

Reply:
It still used for Jenkins test: https://github.com/openucx/ucx/blob/master/contrib/test_jenkins.sh#L1223

Reply:
ok